### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by chanchal-panpaliya**
project live - https://solutionground.netlify.app/

### CODE REVIEW

File: Lists/List/List.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+.Notification-card{
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  background-color: var(--white);
+  box-shadow: 5px 5px 20px 0 rgb(0 0 0 / 20%);
+  border-radius:15px 0px 0px 5px;
+
+  display: flex;
+  flex-direction:row;
```
Comment (Mohasin-Haque) [inline Lists/List/List.css:17]: You can make a class for flex and use

File: Lists/index.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+:root {
+  --icon-primary-color:#594adb ;
+  --icon-secondary-color: #f17c0e;
+    /* --icon-primary-color:#5468ff; darkblue*/
+      /* --icon-primary-color:#990000 ; 
+  --icon-secondary-color: #FFCC00; */ 
+
+  /* --icon-secondary-color : #7264e8; #f17c0e*/
```
Comment (Mohasin-Haque) [inline Lists/index.css]: remove comments

File: Lists/utility/comman.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  .sidebar-onhover-primary:hover{
+    color: var(--icon-primary-color);
+  }
+
+  .codespan{
+    background-color: var(--secondary-background-color);
+    color: var(--icon-primary-color);
+  }
+
+  /*----------------------*/
```
Comment (Mohasin-Haque) [inline Lists/utility/comman.css]: Why this ??

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is god but it needs to be formatted and clean. It will be good if you remove the whitespaces and add meaningful comments!!

File: Lists/List/List.css (hunk 1 above)
Comment (chanchal-panpaliya) [reply Lists/List/List.css:17]: I already have a flex utility class. but I don't want to use it here.

File: Lists/List/List.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+.border-weight-left-tertiarycolor{
+    border-style: solid;
+    border-width: 0px 0px 0px 1rem;
+    border-color:var(--tertiary-color)
+}
+
+.border-weight-left-forth-color{
+    border-style: solid;
+    border-width: 0px 0px 0px 1rem;
+    border-color:var(--forth-color)
```
Comment (SanjayJatti) [inline Lists/List/List.css:48]: give better naming convention for border-color. --fourth color does not tell anything.

File: Lists/List/List.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+.notification{
+    width: 100%
+}
+
+.Notification-card{
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  background-color: var(--white);
+  box-shadow: 5px 5px 1.25rem 0 rgb(0 0 0 / 20%);
```
Comment (SanjayJatti) [inline Lists/List/List.css:14]: You can create variable for box shadow also.

**Review by SanjayJatti — APPROVED** [review]

**Pull request merged**